content/pt-br/docs/reference/glossary/garbage-collection.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Coletor de lixo
+id: garbage-collection
+date: 2021-07-07
+full_link: /docs/concepts/architecture/garbage-collection/
+short_description: >
+  Um termo coletivo para os vários mecanismos que o Kubernetes usa para limpar os recursos do cluster.
+
+aka: 
+tags:
+- fundamental
+- operation
+---
+
+Coletor de lixo (Garbage collection) é um termo coletivo para os vários mecanismos que o Kubernetes usa para limpar os recursos do cluster.
```

Comment:
Can you complement this as it is in English?

"Garbage collection is a collective term for the various mechanisms Kubernetes uses to clean up cluster resources.[[-]](javascript:void(0))
Kubernetes uses garbage collection to clean up resources like [unused containers and images](https://kubernetes.io/docs/concepts/architecture/garbage-collection/#containers-images), [failed Pods](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection), [objects owned by the targeted resource](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/), [completed Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/), and resources that have expired or failed.
"

This is due it is not clear that GC is just for expired objects :D 

Thanks!

Reply:
> Can you complement this as it is in English?
> 
> "Garbage collection is a collective term for the various mechanisms Kubernetes uses to clean up cluster resources.[-] Kubernetes uses garbage collection to clean up resources like [unused containers and images](https://kubernetes.io/docs/concepts/architecture/garbage-collection/#containers-images), [failed Pods](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection), [objects owned by the targeted resource](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/), [completed Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/), and resources that have expired or failed. "
> 
> This is due it is not clear that GC is just for expired objects :D
> 
> Thanks!

Sorry, I didn't understand. This text is on line 19.

Reply:
nit: "garbage collection" refers to the act of removing garbage and not to the software itself. So I think "coleta de lixo" would be a better translation instead of "coletor de lixo".

Reply:
Hi @MrErlison and @stormqueen1990 , I agree with @rikatz, here in Brazil, the best terms to our public understand that concept is "Coletor de lixo".

Please @rikatz, can you approve your request changes?

OBS: @rikatz let us know, all your requests was attend?

Thanks

Reply:
@jcjesus the main reason why I don't feel like "coletor de lixo" is a good translation is because this glossary entry refers to the act of collecting unused objects, rather than to the agent that performs said collection (the garbage collector itself).

Reply:
Hi @jcjesus! Good to see you back.

I'm familiar with the term "garbage collector / coletor de lixo", but on the other hand I agree with @stormqueen1990 explanation.

Reply:
@stormqueen1990 @MrErlison 
The collector is the agent, therefore in portuguese the actor responsible to cleaner or the agent should be "coletor" it´s simple.

We need keep below in our mind:
The job translation is about the portuguese community, they should be our target and not our personal opinion.

I´ll keep my request opned, to translate to "coletor".
Thanks

Reply:
> Hi @jcjesus! Good to see you back.

Thanks @MrErlison, I will try keep looking here and helping our community.
Thanks for your effort to translate the k8s docs.

Reply:
@jcjesus
I understand that. But the original glossary entry doesn't refer to the agent. It refers to the act of cleaning unused resources.